website/docs/r/redhat_openshift_cluster.html.markdown

```diff
@@ -217,6 +217,8 @@ A `network_profile` block supports the following:
 
 * `outbound_type` - (Optional) The outbound (egress) routing method. Possible values are `Loadbalancer` and `UserDefinedRouting`. Defaults to `Loadbalancer`. Changing this forces a new resource to be created.
 
+* `preconfigured_nsg_enabled` - (Optional) Whether a preconfigured network security group is being used on the subnets.  Defaults to `false`.  Changing this forces a new resource to be created.
```

Comment:
(as above)

```suggestion
* `preconfigured_network_security_group_enabled` - (Optional) Whether a preconfigured network security group is being used on the subnets.  Defaults to `false`.  Changing this forces a new resource to be created.
```

Reply:
Apologies, I missed the fact that you made this change and I simply needed to commit.  Regardless, I fixed in commit becf28602323ab08a37def639367eb0975a74629